ui/storybook/README.md

```diff
@@ -0,0 +1,114 @@
+# Storybook
+
+This package is used to generate documentation for Perses UI components using [Storybook](https://storybook.js.org/).
+
+## Getting started
+
+- Run storybook in development mode: `npm run storybook`
+
+## Creating stories
+
+### Where should stories live?
+
+Stories **SHOULD** live in the first entry in the list below that matches their purpose:
+
+- Stories that are tightly coupled with a component should be colocated with the component code in the associated package (e.g. the stories for the `LineChart` component live at `ui/components/src/LineChart/LineChart.stories.tsx` as a sibling of `LineChart.tsx`).
+- Package-specific stories should live in the `src/stories` directory for that package.
+- Stories that provide project-wide documentation or involve components from multiple packages should live in the storybook package (`ui/storybook/stories`).
+
+### What kind of story format should I use?
+
+#### Components
+
+Stories for components should use [Component Story Format (CSF)](https://storybook.js.org/docs/react/api/csf) and follow the `PascalCaseName.stories.tsx` naming scheme.
+
+See [Writing stories in TypeScript](https://storybook.js.org/blog/writing-stories-in-typescript/) for some additional guidance on typing for stories.
+
+#### Text-only documentation
+
+Stories that are primarily text-based documentation (e.g. the `About` pages for each package) should be written using [MDX](https://storybook.js.org/docs/react/writing-docs/mdx) and follow the `PascalCase.stories.mdx` naming scheme.
+
+These may include importing a few small components like the `LinkTo` component from the [Link addon](https://storybook.js.org/addons/@storybook/addon-links/) to link out to other stories.
+
+### Importing package code
```

Comment:
This section describes one of the most controversial things I'm doing in this setup. Please let me know if you really dislike these tradeoffs and have suggestions for other approaches.